### PR TITLE
:sparkles: add search bar and all skils

### LINKED ISSUE #296: In Admin page, able to see all skills
Should be nice to get another section on the admin page with all skills and be able to modify any skills (like name, description, category..)

### PR DESCRIPTION
**Pull request by MailyLehoux**
resolve #296 #297

### CODE REVIEW

File: src/pages/admin.tsx [hunk 1]
```diff
@@ -36,37 +52,69 @@ export default function AdminPage() {
...
+          ) : (
+            <div>
+              <div className="flex flex-col mb-8 mt-10">
+                <h1 className="text-xl">{t("admin.skillsPending")}</h1>
+                {skills.Skill && (
+                  <p className="opacity-50">
+                    {
+                      skills.Skill.filter((field) => field.verified === false)
+                        .length
+                    }{" "}
```
Comment (benjaminbrgs) [inline src/pages/admin.tsx:77]:
```suggestion
                    }
```

File: src/pages/admin.tsx [hunk 2]
```diff
@@ -36,37 +52,69 @@ export default function AdminPage() {
...
-            )}
+          <SearchBar
+            initialValue={search}
+            value={search}
+            setSearch={setSearch}
+            placeholder={t("search.placeholder")}
+          />
+          {loading ? (
+            <div />
+          ) : (
```
Comment (benjaminbrgs) [inline src/pages/admin.tsx:68]:
```suggestion
          {!loading && (
```

I suggest this to avoid unused div

**Review by benjaminbrgs — CHANGES_REQUESTED** [review]
Nice !
The admin section becomes very useful 💪🏼

File: src/pages/admin.tsx (hunk 1 above)
Comment (MailyLehoux) [reply src/pages/admin.tsx:77]: Personnally, I prefer with a white space. In search section it's also like this

Comment (benjaminbrgs) [reply src/pages/admin.tsx:77]: True sorry! I thought it was a trailing space

**Review by benjaminbrgs — APPROVED** [review]

**Pull request merged**